.github/workflows/cd-workflow.yml

```
@@ -0,0 +1,93 @@
+          distribution: 'temurin'
+          server-id: github
+          setting-path: ${{ github.workspace }}
+
+      - name: 👏🏻 grant execute permission for gradlew
+        run: chmod +x gradlew
+
+      - name: 🐘 build with Gradle (without test)
+        run: ./gradlew clean build -x test --stacktrace
+
+      - name: 🐳 Docker image build & push
+        run:  |
```

YAMLlint (1.37.1)

[warning] too many spaces after colon

(colons)